demo/components/LiveEdit.js

```diff
@@ -17,21 +17,13 @@ const StyledProvider = styled(LiveProvider)`
   margin-bottom: ${polished.rem(100)};
 `
 
-const LiveWrapper = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: stretch;
-  align-items: stretch;
-
-  @media (max-width: 600px) {
-    flex-direction: column;
-  }
-`
 
 const column = css`
   flex-basis: 50%;
   width: 50%;
   max-width: 50%;
+  display:inline;
```

Comment:
Sorry, I meant the other way around. This is encapsulated by a flex element, so the idea is to remove the `inline` and `float` CSS snippets